Python/sysmodule.c

```diff
@@ -2923,7 +2923,11 @@ _PySys_InitMain(PyThreadState *tstate)
     SET_SYS_FROM_WSTR("exec_prefix", config->exec_prefix);
     SET_SYS_FROM_WSTR("base_exec_prefix", config->base_exec_prefix);
     {
-        PyObject *str = PyUnicode_FromString(PLATLIBDIR);
+        const char* platlibdir = getenv("_PYTHON_PLATLIBDIR");
```

Comment:
Why "_PYTHON_PLATLIBDIR" name, rather than PYTHONPLATLIBDIR?

You should use _Py_GetEnv(config->use_environment, "PYTHONPLATLIBDIR") if your environment variable name  starts with "PYTHON".